src/main/java/org/candlepin/dto/api/v1/ConsumerTranslator.java

```diff
@@ -206,7 +215,7 @@ public ConsumerDTO populate(ModelTranslator translator, Consumer source, Consume
         else {
             dest.setReleaseVer(null);
             dest.setOwner(null);
-            dest.setEnvironment(null);
+            dest.setEnvironments(Collections.emptyList());
```

Comment:
_If we're going to standardize around null values_ for no environments, we should also make it null here

Reply:
Done.